test-resources/infra/sts-mock/token/api.yaml

```
@@ -45,7 +45,7 @@ Resources:
             - !Ref Environment
             - BaseDns
         - !Sub
-          - sts-mock.${DNS_RECORD}
+          - test-resources.${DNS_RECORD}
```

```suggestion
          - sts-mock.${DNS_RECORD}
```

This is still the sts-mock API.